build.gradle

```
@@ -194,6 +211,74 @@ subprojects {
     }
 }
 
+project(":api") {
+    apply plugin: "org.openapi.generator"
+
+    // This project should only contain generated code.  No point in running checkstyle
+    checkstyle.sourceSets = []
+
+    ext {
+        api_spec_path = "${projectDir}/candlepin-api-spec.yaml"
+        config_file = "${projectDir}/candlepin-api-config.json"
+    }
+
+    openApiGenerate {
+      generatorName = "jaxrs-spec"
+      inputSpec = api_spec_path
+      configFile = config_file
+      outputDir = "$buildDir/generated"
+      configOptions = [
+          interfaceOnly: true,
+          generatePom: false
```

```suggestion
          generatePom: false,
          dateLibrary: "java8"
```

Not mandatory or anything, but helpful for future proofing

---

:+1: